File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processes/RootProcessStep.ts
import { GuidValue, LanguageTag } from '@omnia/fx-models';
import { InternalProcessStep } from './InternalProcessStep';

export interface RootProcessStep extends InternalProcessStep {
    enterpriseProperties: { [internalName: string]: any };
    processTemplateId: GuidValue;
    comment: string;
}

File: src/Omnia.ProcessManagement.Models/Processes/RootProcessStep.cs
using Newtonsoft.Json.Linq;
using Omnia.Fx.Models.Language;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.Processes
{
    public class RootProcessStep : ProcessStep
    {
        public Dictionary<string, JToken> EnterpriseProperties { get; set; }
        public Guid ProcessTemplateId { get; set; }
        public string Comment { get; set; }
    }
}
